File: Sakila_Net/CS_SakilaLib/Neg/NegStaff.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegStaff
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOStaff.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClStaff staff) 
        {
            return DAO.DAOStaff.sqlSelect(staff);
        }
        public  static int crear(ClStaff staff)
        {
            return DAO.DAOStaff.sqlInsert(staff);
        }
        public  static int eliminar(ClStaff staff)
        {
            return DAO.DAOStaff.sqlDelete(staff);
        }
        public  static int actualizar(ClStaff staff)
        {
            return DAO.DAOStaff.sqlUpdate(staff);
        }
    }
}

File: Sakila_Net/VB_SakilaLib/Neg/NegStaff.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegStaff
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOStaff.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( staff as ClStaff) as Boolean
                        Return DAO.DAOStaff.SelectOne(staff)     
                    End Function
                    Public  Shared Function crear( staff as ClStaff) as Integer
                        Return DAO.DAOStaff.SqlInsert(staff)     
                    End Function
                    Public  Shared Function eliminar( staff as ClStaff) as Integer
                        Return DAO.DAOStaff.SqlDelete(staff)     
                    End Function
                    Public  Shared Function actualizar( staff as ClStaff) as Integer
                        Return DAO.DAOStaff.SqlUpdate(staff)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
